lib/conversions/test/spec/run-test-suite.hs

```diff
@@ -0,0 +1,13 @@
+module Main where
+
+import Prelude
+
+import Cardano.Startup
+    ( withUtf8Encoding )
+import Test.Hspec.Extra
+    ( hspecMain )
+
+import qualified Spec
+
+main :: IO ()
+main = withUtf8Encoding $ hspecMain Spec.spec
```

Comment:
there is a https://hackage.haskell.org/package/with-utf8-1.0.2.4/docs/Main-Utf8.html#v:withUtf8

I suggest you to use the hackage library `with-utf8` instead of a custom `Cardano.Startup`

Reply:
> there is a https://hackage.haskell.org/package/with-utf8-1.0.2.4/docs/Main-Utf8.html#v:withUtf8
> 
> I suggest you to use the hackage library `with-utf8` instead of a custom `Cardano.Startup`

Hi @Unisay 

Thanks for the suggestion! I'll make a subsequent PR that fixes this. (I think your suggestion can be applied to several packages.)

Reply:
@Unisay 
As promised: https://github.com/cardano-foundation/cardano-wallet/pull/4157